dracut/30ignition/sysroot-boot.service

```diff
@@ -11,6 +11,10 @@ RequiresMountsFor=/sysroot
 Requires=initrd-setup-root.service
 After=initrd-setup-root.service
 
+# This is to make sure that /sysroot/boot is unmounted before we start
+# switching root in initrd-switch-root.target.
+After=initrd-switch-root.target
```

Comment:
Isn't it a bit confusing to have this be `After` instead of `Before`? Both would do the fix, right? Then I would prefer `Before`.

Reply:
Yeah, for conflicting targets, it does not matter whether it's `After` or `Before`, because systemd orders stop jobs before start jobs. But will update.